File: ToDoList/Front-End/ToDoListFront/src/app/models/AssignmentList/assignment-list.ts
import { Guid } from "guid-typescript";
import { AssignmentDto } from "../Assignment/AssignmentDto"; 
import { AssignmentListTypeDto } from "../AssignmentListTypeDto";
import { User } from "../User/user";

export class AssignmentList{
    public id?: Guid;
    public name?: string;
    public status?: boolean;
    public users?: User[];
    public listType?: AssignmentListTypeDto;
    public assignment?: AssignmentDto[];
}

File: ToDoList/Back-End/ToDoList.Data/Entities/AssignmentList.cs
namespace ToDoList.Data.Entities;
public class AssignmentList {
    public Guid Id { get; set; }
    public string Name { get; set; }
    public bool Status { get; set; }
    public ICollection<User> Users { get; set; }
    public ListType ListType { get; set; }
    public ICollection<Assignment> Assignments { get; set; }
}

